private/buf/cmd/buf/command/alpha/plugin/pluginpush/pluginpush.go

```diff
@@ -300,7 +300,7 @@ func run(
 		bufplugin.PluginToProtoPluginRegistryType(plugin),
 		plugin.Version(),
 		plugin.ContainerImageDigest(),
-		bufplugin.PluginOptionsToOptionsSlice(plugin.DefaultOptions()),
+		bufpluginconfig.PluginOptionsToOptionsSlice(plugin.DefaultOptions()),
```

Comment:
Moved these utility methods to `bufpluginconfig` to avoid circular dependencies. They make more sense to live there since that's where we convert `[]string` options to/from `map[string]string`.